lib/components/form/ViewInverters.tsx

```diff
@@ -0,0 +1,152 @@
+import { FC, useState } from 'react';
+import { InverterCard } from '../InverterCard';
+import Button from '../Button';
+import useSWR from 'swr';
+import useSWRMutation from 'swr/mutation';
+import { InverterPutData, Inverters } from '~/lib/types';
+import { PlusCircleIcon } from '@heroicons/react/24/outline';
+import Link from 'next/link';
+import { getAuthenticatedRequestOptions } from '~/lib/swr';
+import { ChevronLeftIcon } from '@heroicons/react/24/solid';
+import Spinner from '../Spinner';
+
+async function sendRequest(url: string, { arg }: { arg: InverterPutData }) {
+  const options = await getAuthenticatedRequestOptions(url);
+  return fetch(url, {
+    method: 'PUT',
+    body: JSON.stringify(arg),
+    credentials: 'include',
+    headers: {
+      'Content-Type': 'application/json',
+      ...options.headers,
+    },
+  });
+}
```

Comment:
Do we want to move this function into some other file? Seems re-usable

Reply:
Yeah, it does seem reusable, but I don't really want to spend time to deal with passing in different types for `arg` right now...